torch/package/package_importer.py

```diff
@@ -208,6 +208,9 @@ def persistent_load(saved_id):
                 storage = loaded_storages[key]
                 return storage
             elif typename == "reduce_package":
+                if len(data) == 2:
```

Comment:
maybe add a comment that this is for BC, and causes produced objects not to be shared properly